params/stories/create.go

```diff
@@ -0,0 +1,20 @@
+package storyparams
+
+import "github.com/source-academy/stories-backend/model"
+
+type Create struct {
+	AuthorID     uint   `json:"authorId"`
+	StoryContent string `json:"storyContent"`
```

Comment:
Although this need not be the same field name, what necessitates that this field be called `StoryContent` instead of just `Content`?